src/analytics/lambdas/custom-resource/create-schemas.ts

```diff
@@ -323,6 +327,10 @@ function getCreateOrUpdateViewForReportingSQL(newAddedAppIdList: string[], props
       table_event_parameter: odsTableNames.event_parameter,
```

Comment:
Should we remove these legacy tables?

Reply:
sp-migrate-ods-events-1.0-to-1.1.sql still use it.

Reply:
Let's remove the sql too. It does not make sense to keep it in the latest version.

Reply:
refactored it